libs/sdk-ui-dashboard/src/model/events/index.ts

```diff
@@ -494,7 +497,9 @@ export type DashboardEvents =
     | DashboardDrillToInsightRequested
     | DashboardDrillToLegacyDashboardRequested
     | DashboardDrillRequested
-    | DashboardDrillableItemsChanged;
+    | DashboardDrillableItemsChanged
+    // internal
+    | CreateInsightRequested;
```

Comment:
What about handle it by `userInteractionTriggered` instead of the new event? It will be even more hidden there :)

Reply:
@kandl Indeed I was thinking about it, but it seems to me like a misusing of it, but if you think it will be better...

Reply:
I will not abuse interaction ... new one seems legit